crossplane.yaml

```diff
@@ -47,47 +47,9 @@ metadata:
     friendly-kind-name.meta.crossplane.io/services.aws.platformref.crossplane.io: Cluster Services
     friendly-kind-name.meta.crossplane.io/eks.aws.platformref.crossplane.io: EKS Cluster
 
-    # TODO(negz): Remove these older style annotations once annotation consumers
-    # have had a chance to update.
-    company: Upbound
-    maintainer: Jared Watts <jared@upbound.io>
-    keywords: aws, cloud-native, kubernetes, example, platform, reference
-    source: github.com/upbound/platform-ref-aws
-    license: Apache-2.0
-    descriptionShort: |
-      The AWS reference platform for Kubernetes and Data Services.
-
-    description: |
-      This reference platform Configuration for Kubernetes and Data Services
-      is a starting point to build, run, and operate your own internal cloud
-      platform and offer a self-service console and API to your internal teams.
-
-    readme: |
```

Comment:
Are all of the fields completely covered by the new annotations style fields now? i'm not sure if `company` is covered still, maybe others?

e.g. does everything look populated and visible when you build your fork of this package and push it to Upbound Cloud Registry to view its entry/details? the upstream one is at https://cloud.upbound.io/registry/upbound/platform-ref-aws, your fork would be something similar after you build/push.

Reply:
That's a great way to test it, thank you for pointing it out. And indeed not all fields are populated in the web UI in my configuration. I will revert the changes. Btw, are the annotations dependent on crossplane version?

Reply:
Annotations reverted and tested in the Web UI. 
![image](https://user-images.githubusercontent.com/8086638/158196912-d8680fd0-652e-4c6f-aae9-f518fb49bbae.png)